stacks/camel/1.0.0/devfile.yaml

```diff
@@ -0,0 +1,81 @@
+schemaVersion: 2.2.0
+metadata:
+  name: camel
+  displayName: Apache Camel Basics
+  description: Gain familiarity with Apache Camel by getting hands-on following documented guidelines.
+  icon: https://raw.githubusercontent.com/apache/camel/refs/heads/main/docs/img/logo-d.svg
+  tags:
+    - Java
+    - Quarkus
+    - Spring
+    - Camel
+  projectType: Camel
+  language: Camel
+  version: 1.0.0
+  website: https://camel.apache.org/
+starterProjects:
+  - name: devsandbox-camel
+    git:
+      checkoutFrom:
+        revision: main
+      remotes:
+        origin: 'https://github.com/RedHat-Middleware-Workshops/devsandbox-camel.git'
+components:
+  - name: tools
+    container:
+      image: quay.io/devfile/universal-developer-image:ubi8-304d6d4
+      memoryLimit: 3Gi
+      env:
+        - name: JAVA_HOME_TEST
+          value: /home/tooling/.sdkman/candidates/java/17.0.3-tem
```

Comment:
Is it possible to get rid of explicit version of the JDK?
if not it means that when updating the container image, the version might need to be adjusted

There are already several environment variable available pointing to the JDk from the image itself: https://github.com/devfile/developer-images?tab=readme-ov-file#java

Reply:
Done. It was a residual variable I used in the past.